src/Discord.Net.Webhook/DiscordWebhookClient.cs

```diff
@@ -91,6 +91,37 @@ private static API.DiscordRestApiClient CreateApiClient(DiscordRestConfig config
             string username = null, string avatarUrl = null, RequestOptions options = null, AllowedMentions allowedMentions = null)
             => WebhookClientHelper.SendMessageAsync(this, text, isTTS, embeds, username, avatarUrl, allowedMentions, options);
 
+        /// <summary> Modifies a message posted using this webhook. </summary>
+        /// <remarks>
+        ///     This method can only modify messages that were sent using the same webhook.
+        /// </remarks>
+        /// <param name="messageId">ID of the modified message.</param>
+        /// <param name="text">The message to be sent.</param>
+        /// <param name="embeds">The <see cref="Discord.EmbedType.Rich" /> <see cref="Embed" /> enumeration to be sent.</param>
+        /// <param name="allowedMentions">
+        ///     Specifies if notifications are sent for mentioned users and roles in the message <paramref name="text"/>.
+        ///     If <c>null</c>, all mentioned roles and users will be notified.
+        /// </param>
+        /// <param name="options">The options to be used when sending the request.</param>
+        /// <returns>
+        ///     A task that represents the asynchronous modification operation.
+        /// </returns>
+        public Task ModifyMessageAsync(ulong messageId, string text = null, IEnumerable<Embed> embeds = null,
```

Comment:
Actually, this needs to be changed, follow the same pattern as modifying user messages.
See the following: https://github.com/discord-net/Discord.Net/blob/dev/src/Discord.Net.WebSocket/Entities/Messages/SocketUserMessage.cs#L166-L167

They need to optionally change what they want, a simple "null" isn't able to tell if they want to clear it or not send at all.